### PR TITLE
Dspau

### PR DESCRIPTION
**Pull request by iliaplatone**

### CODE REVIEW

**Comment by iliaplatone** [comment]
Maybe was removed in the last commit..
I must check. If you can fix it please remove it. 


---- On mar, 29 gen 2019 09:29:17 +0100 notifications@github.com wrote ----

I see git submodules still in the diff?

File: .circleci/build-3rdparty.sh [hunk 1]
```diff
@@ -4,6 +4,8 @@ set -x -e
 
 FLAGS="-DCMAKE_INSTALL_PREFIX=/usr/local -DFIX_WARNINGS=ON -DCMAKE_BUILD_TYPE=$1"
 
+modules=$(grep path .gitmodules | grep 3rdparty | cut -d '=' -f 2 | tr -d '\n')
```
Comment (knro) [inline .circleci/build-3rdparty.sh]: What is this needed for now?

File: .circleci/build-all.sh [hunk 2]
```diff
@@ -5,6 +5,14 @@ set -x -e
 br=( master develop package travis pull )
 
 build_all () {
+    modules=$(grep path .gitmodules | cut -d '=' -f 2 | tr -d '\n')
```
Comment (knro) [inline .circleci/build-all.sh]: Not required.

File: .circleci/build-all.sh [hunk 3]
```diff
@@ -5,6 +5,14 @@ set -x -e
...
+
+    for module in $modules; do
```
Comment (knro) [inline .circleci/build-all.sh]: Same

File: travis-ci/deps-linux.sh [hunk 4]
```diff
@@ -36,6 +36,14 @@ sudo apt-get -q -y install \
  libswscale-dev \
  libavutil-dev \
 
+modules=$(grep path .gitmodules | cut -d '=' -f 2 | tr -d '\n')
```
Comment (knro) [inline travis-ci/deps-linux.sh]: Not required.

**Pull request closed without merging**